modules/tinymce/src/plugins/visualchars/test/ts/browser/PluginTest.ts

```diff
@@ -56,6 +56,7 @@ describe('browser.tinymce.plugins.visualchars.PluginTest', () => {
     editor.setContent('<p>abc&nbsp;&nbsp;</p>');
 
     TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 3);
+    editor.focus();
```

Comment:
I'm sure this is fine, but it would probably be better to set focus right after instantiating the hook.

Reply:
Focus is only required for the `setRng` method though, I felt this was more explicit. From my testing the test behaviours between setting focus right after instantiating the hook and placing it right before `setRng` are the same.

Reply:
Maybe move the focus call to the start of the test function or use focusOnInit on the hook since the other tests doesn't seem to depend on focus being there since they all block on toolbars buttons first.

Reply:
Yeah fair, thinking about it using focusOnInit would probably be more elegant. Mitch has also mentioned that before.